### ISSUE TITLE
Taxonomy Support

### ISSUE THREAD

**Issue by roikles** [roikles/Flexbones#49]
We have `libs/cpts` and I think that lib/taxonomies should be added with a boilerplate taxonomy creation template.

**Comment by DrizzlyOwl** [comment]
Agreed. 

Would it be beneficial to include code for attached 'categories' and 'post_tags' functionality into CPTs?

**Comment by roikles** [comment]
Something like this?

```php
<?php

$single_name = "Book";
$plural_name = "Books";

$labels = array(
    'name' => $plural_name,
    'singular_name'  => $single_name,
    'search_items' => 'Search ' . $plural_name,
    'all_items' => 'All ' . $plural_name,
    'parent_item' => 'Parent ' . $single_name,
    'parent_item_colon' => 'Parent ' . $single_name,
    'edit_item' => 'Edit ' . $single_name,
    'update_item' => 'Update ' . $single_name,
    'add_new_item' => 'Add New ' . $single_name,
    'new_item_name' => 'New '. $single_name . ' Name' ,
    'menu_name' => $plural_name,
);

$args = array(
    'hierarchical' => true,
    'labels' => $labels,
    'show_ui' => true,
    'show_admin_column' => true,
    'query_var' => true,
    'rewrite' => array( 'slug' => 'gallery' ),
);

register_taxonomy( 'genre', array( 'projects' ), $args );

```

**Comment by roikles** [comment]
Fixed in 0d5eadf
